src/Components/PetCard.js

```diff
@@ -10,7 +10,7 @@ const PetCard = ({ id, name, image }) => {
   return (
     <div className="pet-card">
       <Link style={linkStyle} to={`pet/${id}`}>
-        <img className="pet-image" src={image} alt="{name}"/>
```

Comment:
Oops! Good catch getting rid of those " ". Our test was failing getting that alt tag till these were removed.